examples/twittermap/web/public/javascripts/searchbar/controllers.js

```diff
@@ -1,7 +1,36 @@
 angular.module('cloudberry.util', ['cloudberry.common'])
   .controller('SearchCtrl', function($scope, $window, cloudberry, cloudberryConfig, moduleManager) {
       var stopwordsMap = buildStopwordsMap();
+        
+        $("#keyword-textbox").autocomplete({source:[]});
+        $("#keyword-textbox").on("keyup",function(event){
+          
+          var q = $scope.keyword;
+          var url = "http://127.0.0.1:5000/spoof?query="+q;
```

Comment:
The `proxy` server should be running on our `Twittermap` server side, then this `127.0.0.1` address will not work. There're 2 options to implement this: (1) make your `ajax` request to a relative path with the same domain as this website is from; (2) make this absolute address configurable in our `application.conf` of `Twittermap` application.